smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/fault/KafkaDeadLetterQueue.java

```diff
@@ -35,6 +35,7 @@
 @SuppressWarnings({ "rawtypes", "unchecked" })
 public class KafkaDeadLetterQueue implements KafkaFailureHandler {
 
+    public static final String DEAD_LETTER_EXCEPTION = "dead-letter-exception";
```

Comment:
I would name it exception-class-name. That would avoid the confusion between this one, the "reason" and the "cause". Actually, adding "cause-class-name" might be useful too.

Reply:
Done